src/client/components/equalizer/upload/infoAboutFile/infoAboutUploadFile.js

```diff
@@ -1,32 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import './infoAboutUploadFile.css';
+import { musicIcon } from '../../../../assets/icons/icons';
 
 const InfoAboutTrack = (props) => {
-  const { trackname, tracksize, tracktype } = props;
+  const { trackname } = props;
```

Comment:
You can destruct this, right from function parameters

Reply:
done)